payjoin-cli/src/main.rs

```diff
@@ -50,10 +54,13 @@ fn cli() -> ArgMatches {
         .subcommand(
             Command::new("send")
                 .arg_required_else_help(true)
-                .arg(arg!(<BIP21> "The `bitcoin:...` payjoin uri to send to"))
+                .arg(
+                    Arg::new("BIP21")
+                    .help("The `bitcoin:...` payjoin uri to send to"))
                 .arg_required_else_help(true)
                 .arg(
-                    arg!(--fee_rate <VALUE>)
+                    Arg::new("fee-rate")
```

Comment:
The arg ID here is inconsistent the above "cookie_file" and "cookie-file". I'd choose the former for ID and latter for argument in the CLI